src/backend/db/products.js

```diff
@@ -33,8 +41,11 @@ export const products = [
     price: '80,000',
```

Comment:
instead of two separate variables you can put only one integer and use .toLocaleString() to get the commas in UI